examples/sampler/sampler_example.ipynb

```diff
@@ -20,9 +20,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
Did you mean to check this in?

Reply:
This is resolved in the most recent commit